### ISSUE TITLE
TouchRipples Effect

### ISSUE THREAD

**Issue by Loth4r** [Rajawali/Rajawali#1661]
#### Rajawali Version or Branch
master

#### Device and Android Version
all

#### Summary
Hello there,

Are there any future plans on making TouchRipple effect example working again?

https://github.com/Rajawali/Rajawali/blob/master/examples/src/main/java/org/rajawali3d/examples/examples/effects/TouchRipplesFragment.java

For now I see most of the code commented out - as I understand this is due to 1.1.x release api/naming convention changes. After analysing I can see that there is a crucial class called TouchRippleFilter that seems to be essential for this effect. It does not seem to be implemented anywhere in latest Rajawali library release. Are there plans for doing so?

Thanks in advance

**Comment by ToxicBakery** [comment]
Yes, it was knowingly broken quite a while back I believe but it will be fixed at some point. PRs are welcome if you want to take a run at it.

**Comment by ry-narola** [comment]
But the latest app on playstore contains this effect

**Comment by Loth4r** [comment]
Latest Play store app has been released quite some time ago and it is basing on older version of Library that that this effect working.
